src/utils/model_states.py

```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+
+
+class ModelState(Enum):
+    """
+    Possible states of a mode
+    """
+
+    Ready = 1
+    Building = 2
+    Recovering = 3
+    FailedRecovery = 4
+    Failed = 5
+    Deployed = 6
+    Disabled = 7
+    DisabledRecovery = 8
+    DisabledFailed = 9
+    Deleted = 10
+
+    def __eq__(self, other):
+        if isinstance(other, str):
+            return str(self.name) == other
+        return super().__eq__(other)
+
+    def __str__(self):
+        return self.name
+
+
+k = ModelState['Ready']
+print(f"Model is {k}")
+
+print(k in [ModelState.Ready, ModelState.Building])
+print(k == 'Ready')
```

Comment:
I think you can remove that

Reply:
I was testing the enum class lmao, sorry. Here https://github.com/datarisk-io/mlops-neomaril-codex/pull/50/commits/be7182de4900546a4aa2016735a3a61cb43e1bb6